targets/TARGET_NUVOTON/TARGET_M2351/analogout_api.c

```diff
@@ -0,0 +1,198 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2015-2016 Nuvoton
```

Comment:
Can you fix the year (2018 for this new file) ?

Reply:
Also add SPDX identifier please

Reply:
@0xc0170 I fixed them in https://github.com/ARMmbed/mbed-os/pull/8828/commits/e24a3c09d7decd3ef062384ba3830e9558bc26d6.